frontend/services/milestones/getters.js

```diff
@@ -0,0 +1,21 @@
+const getters = (apiRequest) => {
+  const getClosedMilestones = () => {
+    return apiRequest.get(`/api/milestones`, {
+      params: {
+        isClosed: '1',
+      },
+    });
+  };
+  const getOpenMilestones = () => {
+    return apiRequest.get(`/api/milestones`, {
+      params: {
+        isClosed: '0',
+      },
+    });
+  };
```

Comment:
함수를 하나로 합치고 isClosed를 파라미터로 받으면 어떨까요?
```suggestion
  const getMilestones = ({ isClosed = '0' }) => {
    return apiRequest.get(`/api/milestones`, {
      params: {
        isClosed,
      },
    });
  };
```